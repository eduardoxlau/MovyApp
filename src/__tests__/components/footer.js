/* eslint-disable no-undef */
import renderer from 'react-test-renderer';

import Footer from 'components/footer';

jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
  useLocation: () => ({ pathname: '/' }),
  useHistory: () => ({
    push: jest.fn(),
  }),
}));

describe('<Footer />', () => {
  it('should render snapshot footer', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
