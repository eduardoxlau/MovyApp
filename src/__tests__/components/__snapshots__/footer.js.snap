// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render snapshot footer 1`] = `
<footer
  className="bg-black pb-24"
>
  <div
    className="container mx-auto"
  >
    <div
      className="text-white text-3xl mb-4"
    >
      Questions? Call +1 (408) 600-1722 (USA)
    </div>
    <div
      className="flex flex-wrap md:flex-row text-center md:text-left"
    >
      <div
        className="flex-shrink w-1/2 md:w-1/4 my-4 text-white text-lg"
      >
        <a
          href="./"
        >
          FAQ
        </a>
      </div>
      <div
        className="flex-shrink w-1/2 md:w-1/4 my-4 text-white text-lg"
      >
        <a
          href="./"
        >
          Help Center
        </a>
      </div>
      <div
        className="flex-shrink w-1/2 md:w-1/4 my-4 text-white text-lg"
      >
        <a
          href="./"
        >
          Account
        </a>
      </div>
      <div
        className="flex-shrink w-1/2 md:w-1/4 my-4 text-white text-lg"
      >
        <a
          href="./"
        >
          Media Center
        </a>
      </div>
      <div
        className="flex-shrink w-1/2 md:w-1/4 my-4 text-white text-lg"
      >
        <a
          href="./"
        >
          Privacy
        </a>
      </div>
      <div
        className="flex-shrink w-1/2 md:w-1/4 my-4 text-white text-lg"
      >
        <a
          href="./"
        >
          Jobs
        </a>
      </div>
      <div
        className="flex-shrink w-1/2 md:w-1/4 my-4 text-white text-lg"
      >
        <a
          href="./"
        >
          Ways to Watch
        </a>
      </div>
      <div
        className="flex-shrink w-1/2 md:w-1/4 my-4 text-white text-lg"
      >
        <a
          href="./"
        >
          Terms of use
        </a>
      </div>
      <div
        className="flex-shrink w-1/2 md:w-1/4 my-4 text-white text-lg"
      >
        <a
          href="./"
        >
          Speed Test
        </a>
      </div>
      <div
        className="flex-shrink w-1/2 md:w-1/4 my-4 text-white text-lg"
      >
        <a
          href="./"
        >
          Cookie Preferences
        </a>
      </div>
      <div
        className="flex-shrink w-1/2 md:w-1/4 my-4 text-white text-lg"
      >
        <a
          href="./"
        >
          Corporate Information
        </a>
      </div>
      <div
        className="flex-shrink w-1/2 md:w-1/4 my-4 text-white text-lg"
      >
        <a
          href="./"
        >
          Contact Us
        </a>
      </div>
    </div>
  </div>
</footer>
`;
